src/js/select2/utils.js

```diff
@@ -230,8 +228,14 @@ define([
       return true;
     }
 
-    return ($el.innerHeight() < el.scrollHeight ||
-      $el.innerWidth() < el.scrollWidth);
+    var computed = window.getComputedStyle(el);
+    var paddingHeight = parseInt(computed.paddingTop, 10);
+        paddingHeight += parseInt(computed.paddingBottom, 10);
+    var paddingWidth = parseInt(computed.paddingLeft, 10);
+        paddingWidth += parseInt(computed.paddingRight, 10);
+  
+    return ((el.clientHeight - paddingHeight) < el.scrollHeight ||
+      (el.clientWidth - paddingWidth) < el.scrollWidth);
```

Comment:
Why not just use the `width` and `height` from `getComputedStyle`?

```suggestion
    var computedEl = window.getComputedStyle(el);
 
    return (parseFloat(computedEl.height) < el.scrollHeight ||
      parseFloat(computedEl.width) < el.scrollWidth);
```